man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -120,14 +130,16 @@ piadm(1M) -- Manage SmartOS Platform Images
 
     piadm(1M) commands and options are:
 
-      piadm activate <PI-stamp> [ZFS-pool-name]
-      piadm assign <PI-stamp> [ZFS-pool-name]
+      piadm activate <PI-stamp/ipxe> [ZFS-pool-name]
+      piadm assign <PI-stamp/ipxe> [ZFS-pool-name]
 
         Activate a Platform Image for the next boot, on a specified ZFS pool
         if there are more than one bootable pools imported.  It is up to the
         administrator to know which pool the system will actually boot.  If a
         boot image with the specified PI-stamp is unavailable, a warning will
-        be issued but the new PI will be activated anyway.
+        be issued but the new PI will be activated anyway.  If "ipxe" is
+        specified, the pool will boot the system into iPXE instead of
```

Comment:
Should this be something like "loader will default to ipxe first, then fall back to the selected platform image if there is no response"?